android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -509,9 +515,10 @@ private void setWalletBadgeVisibility(boolean visibile) {
     }
 
     private void maybeShowPendingTransactions() {
-        assert mWalletModel != null;
```

Comment:
Wallet model may be `null` if native initialization has not been completed yet. It's a rare edge case, but it's an acceptable scenario, so I've removed the assertions.